src/endpoints/proto/gen/hpke_pb.ts

```diff
@@ -19,7 +19,14 @@
 /* eslint-disable */
```

Comment:
we shouldn't touch machine generated files.

Reply:
Should we then update lint.sh to ignore them?

Reply:
Done in https://github.com/microsoft/azure-privacy-sandbox-kms/pull/11/commits/c19596e2ee3530eb66d4de78742d2bc0daec1302

Reply:
they already have ignore tags but clearly these tags are ignored :smile:. There is a way to configure lint. I can have a look to that after the JWT work or if you want to look into that, that's ok for me.

Reply:
I had a look into it, looks like the ignore tags are for tools other than prettier, so I think the simplest solution is to ignore files in the `/gen` directory which is also conceptually neat. 

As of https://github.com/microsoft/azure-privacy-sandbox-kms/pull/11/commits/d4393c4df8e66c6e171f42b4edd2195c0b09db91 the linter passes without any changes to the generated files, so I think we're good to merge this if you're happy